src/deploy/functions/args.ts

```diff
@@ -15,6 +15,10 @@ export interface Source {
   functionsSourceV1?: string;
   functionsSourceV2?: string;
 
+  // Filled in the "prepare" phase.
+  functionsSourceV1Hash?: string;
+  functionsSourceV2Hash?: string;
```

Comment:
nit* I see that we are repeating "Filled in the "prepare" phase." line from 14, and could see these new properties bunched together with what we have w/ L15-16.

Reply:
We could, also, create a `functionSource` object with these properties:

source: {
  v1: {
    source: String
    hash: String
  }
}

Just a thought - up to you to decide if this is cleaner in any way.

Reply:
Removed the redundant comment.

As for `functionSource` object. While I like the object, I'd like to minimize the # of places I touch in this PR.